### PR TITLE
eslint and prettier added

### PR DESCRIPTION
**Pull request by shrijit37**
# Description

This PR introduces ESLint and Prettier to the project for improved code quality and consistency. ESLint helps identify and fix linting issues in JavaScript code, while Prettier ensures consistent formatting across the codebase. This integration aims to streamline the development process and enhance code readability.

Fixes # (issue)

**Snapshots/Videos:** 

## Type of change

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert
- [ ] ✏️ Updated Test Cases

## Added tests?

- [ ] 👍 Yes
- [x] 🙅 No, because they aren't needed
- [ ] 🙋 No, because I need help

## Internationalization Support?

- [ ] 👍 Yes, Added text tokens
- [x] 🙅 No, because they aren't needed
- [ ] 🙋 No, because I need help

**Steps to test the feature:**
1. Install dependencies by running `npm install --legacy-deps-peer` or `yarn install`.
2. Run ESLint to check for linting and format issues : `npm run lint`.

**Snapshot of the test-cases that are passing:** 
(N/A for linting and formatting)

**Does this PR introduce a breaking change?** 
No breaking changes are introduced. All existing functionalities remain intact, but code style and quality will now be enforced.

"By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice."

### CODE REVIEW

File: hooks/useScale.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-import {create} from 'react-native-pixel-perfect';
+import { create } from 'react-native-pixel-perfect';
```
Comment (Neha) [inline hooks/useScale.ts:4]: There are 2 files which you have linted. Please add commands in the package.json for linting and run the command for whole project.

**Review by Neha — COMMENTED** [review]
Thank you for your work. I have left a few comments, please check.

**Comment by giolaq** [comment]
@shrijit37 Any updates on this? @Neha and me would like to progress on this as it's super important for the health of this project 🙏 Thank you!

**Comment by giolaq** [comment]
This is implemented in #27 now

**Pull request closed without merging**